src/Adapter/File/HtaccessFileGenerator.php

```diff
@@ -52,11 +52,11 @@ class HtaccessFileGenerator
     /**
      * HtaccessFileGenerator constructor.
      *
-     * @param CacheClearer $cacheClearer
+     * @param CacheClearerInterface $cacheClearer
      * @param Tools $tools
      * @param bool $multipleViewsConfiguration
      */
-    public function __construct(CacheClearer $cacheClearer, Tools $tools, $multipleViewsConfiguration)
+    public function __construct(CacheClearerInterface $cacheClearer, Tools $tools, $multipleViewsConfiguration)
```

Comment:
Please define in your BC the change of the signature

Reply:
It's not a BC breaks as the interface still allow the usage of CacheClearer :)